platform/android/java/lib/src/org/godotengine/godot/io/FilePicker.kt

```diff
@@ -145,16 +146,54 @@ internal class FilePicker {
 			if (fileMode != FILE_MODE_OPEN_DIR) {
 				intent.type = "*/*"
 				if (filters.isNotEmpty()) {
-					if (filters.size == 1) {
-						intent.type = filters[0]
+					val resolvedFilters = filters.map { resolveMimeType(it) }.distinct()
+					if (resolvedFilters.size == 1) {
+						intent.type = resolvedFilters[0]
 					} else {
-						intent.putExtra(Intent.EXTRA_MIME_TYPES, filters)
+						intent.putExtra(Intent.EXTRA_MIME_TYPES, resolvedFilters.toTypedArray())
 					}
 				}
 				intent.addCategory(Intent.CATEGORY_OPENABLE)
 			}
 			intent.putExtra(Intent.EXTRA_LOCAL_ONLY, true)
 			activity?.startActivityForResult(intent, FILE_PICKER_REQUEST)
 		}
+
+		/**
+		 * Retrieves the MIME type for a given file extension.
+		 *
+		 * @param p_input the extension whose MIME type is to be determined.
+		 * @return the MIME type as a string, or "application/octet-stream" if the type is unknown.
+		 */
+		private fun resolveMimeType(p_input: String): String {
```

Comment:
`p_input` does not match the java/kotlin stype, can you update the parameter name.

Reply:
I've observed this format being used in other Java/Kotlin files as well, and most importantly, I couldn't come up with a better name 😅.